puppet/zulip/files/postgresql/process_fts_updates

```diff
@@ -43,6 +43,8 @@ sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "../.
 try:
     os.environ['DJANGO_SETTINGS_MODULE'] = 'zproject.settings'
     from django.conf import settings
+    if settings.USING_PGROONGA:
+        exit()
```

Comment:
Thanks for the information.

PGroonga has "newer data are more important" policy. So PGroonga's index update is faster than other full-text search implementation such as GIN. But it's slower than "do nothing".

I hope that sent messages are searchable as soon as possible. (It also respects PGroonga policy.)

But this is Zulip project. Zulip project prefers deferring index update, we should do. It should be done in this pull request? Can we separate it as another issue? Because the current implementation works.